src/Pages/AddNewTask/AddNewTask.tsx

```diff
@@ -0,0 +1,152 @@
+import { AuthContext } from '@/Context/AuthContext';
+import { ToastContext } from '@/Context/ToastContext';
+import { INewTask } from '@/Interfaces';
+import baseUrl from '@/utils/Custom/Custom';
+import UseAuthenticatedQuery from '@/utils/Hooks/UseAuthenticatedQuery';
+import { useContext, useEffect, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { MdKeyboardArrowLeft } from "react-icons/md";
+import { Link, useNavigate } from 'react-router-dom';
+import './AddNewTask.module.scss';
+import { ProjectContext } from '@/Context/ProjectContext';
+import { LoadingIcon } from '@/Components';
+const AddNewTask = () => {
+
+  const required = "This Field is required"
+  const { handleSubmit, register, formState: { errors } } = useForm<INewTask>()
+  const [Loading, setLoading] = useState(false)
+  const navigate = useNavigate()
+
+  const toast = useContext(ToastContext)
+  const data = useContext(AuthContext)
+  let headers = data?.header
+  //!  Add New Task
+  const handleAdd = (data: INewTask) => {
+    console.log(data);
```

Comment:
logs should be removed or commented before merging to the main branch, to avoid revealing sensitive data to users.